js/classes/game_engine.js

```diff
@@ -125,14 +125,21 @@ class GameEngine {
     startInput() {
 
         CANVAS.addEventListener("mousedown", (e) => {
-            this.user.firing = false;
-            this.user.aiming = true;
+            if (e.button === 0) {
```

Comment:
What is this "e.button"?

Reply:
The `button` property of the event specifies which mouse button was pressed/released. I believe 0 is left mouse and 2 is right mouse.